### PR TITLE
Feat/local execution

### PR DESCRIPTION
**Pull request by slpopejoy**
Done, but currently hanging after the first request ...

### CODE REVIEW

File: src/Chainweb/Pact/PactService.hs [hunk 1]
```diff
@@ -175,7 +175,7 @@ initPactService' ver cid chainwebLogger spv act = do
     let !pd = P.PublicData def (unsafeGetChainId cid) def def
     let !pse = PactServiceEnv Nothing checkpointEnv spv pd
 
-    evalStateT (runReaderT act pse) theState
+    evalStateT (runReaderT act pse) (PactServiceState theState Nothing)
```
Comment (larskuhtz) [inline src/Chainweb/Pact/PactService.hs]: This uses the lazy state monad. Is this intentional? Could it increase the risk of space leaks?

Comment (slpopejoy) [reply src/Chainweb/Pact/PactService.hs]: @marklnichols any comment here -- both `PactService` and `Chainweb.Pact.Types` use lazy state. Was there a reason for this? I'm assuming it was an oversight.

Fixing.

Comment (slpopejoy) [reply src/Chainweb/Pact/PactService.hs]: Great catch @larskuhtz

Comment (slpopejoy) [reply src/Chainweb/Pact/PactService.hs]: Note doing strict state across the board @emmanueldenloye @marklnichols @emilypi . 

Also cleaning up some other warnings -- uninitialized variables in tests, and Lars' `putLog` (underscoring).

File: node/Utils/Logging.hs [hunk 2]
```diff
@@ -507,7 +507,7 @@ withElasticsearchBackend mgr esServer ixName inner = do
         , HTTP.requestHeaders = [("content-type", "application/json")]
         }
 
-    putLog a = HTTP.defaultRequest
+    _putLog a = HTTP.defaultRequest
```
Comment (larskuhtz) [inline node/Utils/Logging.hs]: 👍 Thanks @slpopejoy

**Pull request merged**